src/main/scala/codacy/plugins/test/multiple/MultipleTests.scala

```diff
@@ -0,0 +1,87 @@
+package codacy.plugins.test.multiple
+
+import codacy.plugins.test._
+
+import com.codacy.analysis.core.model.{CodacyCfg, Configuration, FileCfg, FileError, FullLocation, Issue, LineLocation}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)